.github/workflows/deploy_branch_preview.yml

```diff
@@ -37,6 +37,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Fetch Tags
+        run: |
+          git fetch --prune --unshallow --tags
```

Comment:
Hmm do we really need this?

Reply:
Yes by default checkout v2 doesnt fetch the git tags so we cant find the latsest tag in the tree, i tried to fetch the tags in the best way possible